Source/SIMPLib/Filtering/ComparisonInputs.h

```diff
@@ -97,18 +97,16 @@ typedef struct
 class SIMPLib_EXPORT ComparisonInputs : public QObject
 {
   Q_OBJECT
-
+  // clang-format off
   PYB11_CREATE_BINDINGS(ComparisonInputs)
-
   PYB11_CREATION()  
-
   PYB11_PROPERTY(QVector<ComparisonInput_t> Inputs READ getInputs)
-
   PYB11_METHOD(void addInput OVERLOAD const.QString,dataContainerName const.QString,attributeMatrixName const.QString,arrayName int,compOperator double,compValue)
   PYB11_METHOD(void addInput OVERLOAD const.ComparisonInput_t.&,input)
```

Comment:
I believe you need a space before `input`.

Reply:
Nope.